custom_components/mammotion/button.py

```diff
@@ -25,11 +25,13 @@ class MammotionButtonSensorEntityDescription(ButtonEntityDescription):
         key="start_map_sync",
         press_fn=lambda coordinator: coordinator.async_sync_maps(),
         entity_category=EntityCategory.CONFIG,
+        entity_registry_enabled_default=False,
```

Comment:
Due to the nature of why these buttons exist, I don't think they should be disabled by default. However debate is encouraged. If we are going to disable these by default, better documentation needs to follow. I'd also argue that RSSI values should be disabled as well and again documentation explaining how to enable just one.

ideally in future I can nuke all those kinds of button actions.